extensions/configuration/issuers.yaml

```diff
@@ -0,0 +1,187 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: issuers.cert.gardener.cloud
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: issuers.cert.gardener.cloud
+    busola.io/extension: resource
+    busola.io/extension-version: '0.5'
+data:
+  details: |-
+    header:
+      - name: spec.acme.email
+        source: spec.acme.email
+      - source: 'status.state ? status.state : "UNKNOWN"'
+        highlights:
+          informative:
+          - UNKNOWN
+        name: status
+        widget: Badge
+        description: status.message
+      - name: spec.acme.server
+        source: spec.acme.server
+    body:
+      - widget: Columns
+        children:
+        - name: included.domains
+          widget: Table
+          showHeader: false
+          source: spec.acme.domains.include
+          children:
+            - source: $item
+              search: true
+        - name: excluded.domains
+          widget: Table
+          showHeader: false
+          source: spec.acme.domains.exclude
+          children:
+            - source: $item
+              search: true
+  form: |-
+    - var: IssuerType
+      dynamicValue: | 
+        $exists(spec.acme) ? 'ACME' : $exists(spec.ca) ? 'CA' : ''
```

Comment:
probably it doesn't work, because if I provide VALID YAML to YAML section I can't create Issuer
Examples:
```
apiVersion: cert-manager.io/v1alpha2
kind: ClusterIssuer
metadata:
  name: letsencrypt-staging
spec:
  acme:
    # You must replace this email address with your own.
    # Let's Encrypt will use this to contact you about expiring
    # certificates, and issues related to your account.
    email: raspberry-pi@home.pi
    server: https://acme-staging-v02.api.letsencrypt.org/directory
    privateKeySecretRef:
      # Secret resource that will be used to store the account's private key.
      name: example-issuer-account-key
    # Add a single challenge solver, HTTP01 using nginx
    solvers:
    - http01:
        ingress:
          class: nginx
```
```
apiVersion: cert-manager.io/v1
kind: Issuer
metadata:
  name: ca-issuer
  namespace: mesh-system
spec:
  ca:
    secretName: ca-key-pair
```
check if it's related to this issues: 
https://github.com/kyma-project/kyma-dashboard/issues/51
https://github.com/kyma-project/busola/issues/2247